### PR TITLE
upload artifacts via API (GSI-91)

### PR DESCRIPTION
**Pull request by KerstenBreuer**
```
- established a generic way how to handle events via the file system
- applied to publishing source events and consuming artifacts
- will later also be used for consuming source events and publishing artifacts
- client utility to collect artifacts from the local file system and project them onto the loader API
```

Please note, the CLI for this and all other components will be added in a different PR.

### CODE REVIEW

File: metldata/event_handling.py
Comment (Cito) [inline metldata/event_handling.py:1]: see comments in the other PR

**Review by Cito — APPROVED** [review]
See comments in the other PR. Should have reviewed in different order, sorry.

**Comment by KerstenBreuer** [comment]
Thanks, no it is my bad. I thought we first get this PR in and then review the other. But I requested your review from both of them at the same time.

**Comment by KerstenBreuer** [comment]
Since all changes of this branch are on the branch #13, let's close this branch.

**Pull request closed without merging**